draft-ietf-quic-transport.md

```diff
@@ -678,8 +707,10 @@ A Handshake packet uses long headers with a type value of 0x7D.  It is
 used to carry acknowledgments and cryptographic handshake messages from the
 server and client.
 
-The connection ID field in a Handshake packet contains a connection ID
-that is chosen by the server (see {{connection-id}}).
+The Destination Connection ID field in a Handshake packet contains a connection
+ID that is chosen by the recipient of the packet; the Source Connection ID
```

Comment:
The server's Initial packet also carries a dest connection ID chosen by the client, right?

Reply:
Servers don't send Initial packets.